docs/usage.md

```diff
@@ -16,6 +16,8 @@ The tool is a command line application built with Java. See below for usage inst
 
 # Usage
 
+Make sure you've downloaded the .jar file under Assets [here](https://github.com/datacommonsorg/import/releases/tag/0.1-alpha.1h). Note the path to .jar.
```

Comment:
Let's link to just the releases page (https://github.com/datacommonsorg/import/releases) incase we ever release newer versions. We can ask users to download the latest.